### PR TITLE
[Doc Feature][Feature gaps for O365 connector deprecation][3937634]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/webhooks-and-connectors/feature-gaps-for-O365-connector.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fdf2d6fda6a83e2f01f6497a2e6018461c34e84d/msteams-platform%2Fwebhooks-and-connectors%2Ffeature-gaps-for-O365-connector.md) | 57 | 56 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/76d66b5d-cebe-4798-8397-1c5c52a1a3c7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/deprecation-note.md](https://github.com/MicrosoftDocs/msteams-docs/blob/580c7e1eac2eaf4f3883465efc5567a83f03ceb7/msteams-platform%2Fincludes%2Fdeprecation-note.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9044f0e5-d6f1-4342-8209-0f517bdb33f3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/deprecation-note.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6bf9e2244bdaf3943ef376f5cc6404acd528cbd6/msteams-platform%2Fincludes%2Fdeprecation-note.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e92dad73-93c3-4a48-a723-06d4a8a57c1c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/deprecation-note.md](https://github.com/MicrosoftDocs/msteams-docs/blob/25c7e07f5c4019122833202df6c6fee4f5ece026/msteams-platform%2Fincludes%2Fdeprecation-note.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/58e17b33-c87c-4b5e-8347-1e1b7c7ef20e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/includes/deprecation-note.md [hunk 1]
```diff
@@ -7,3 +7,22 @@
...
 >
 > * For more information about the **When a Teams webhook request is received** trigger, see [Microsoft Teams - Webhook.](/connectors/teams#microsoft-teams-webhook)
+>
+> **If you've already built O365 Connectors**
+>
+> * Create a Power Automate connector: Power Automate enhances the widely used **Workflows** apps in Teams. It's the scalable and secure approach to transmit data programmatically into and out of Teams. If you adopt this method, you can create one or two workflow templates for posting alerts from your product to Teams channels. This approach simplifies user adoption of the new method. For more information, see [Power Automate for enterprise developers, ISVs, and partners.](/power-automate/developer/dev-enterprise-intro)
+>
+> * Update your Teams app: You can enhance your current Teams app. For example, you can enable users to set up proactive messages based on trigger events within your system. For more information, see [how bots can post to channels through proactive messages.](../resources/bot-v3/bot-conversations/bots-conversations.md#proactive-messages)
```
Comment (Wajeed-msft) [inline msteams-platform/includes/deprecation-note.md:15]: This is V3 docs, we should redirect them developer to V4 docs: https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/send-proactive-messages#send-the-message

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by Wajeed-msft** [comment]
@v-bvishnu - Could you please add the reference to tool which can be used to Convert Connector Message Card format to Adaptive Card?

https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/tools/message-card-to-ac-transformation

File: msteams-platform/includes/deprecation-note.md (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/includes/deprecation-note.md:15]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/deprecation-note.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1f1055b95db791ef4472d9cd230749567680ecc5/msteams-platform%2Fincludes%2Fdeprecation-note.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8a588df7-222d-4ff6-b74d-b75363664112) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/deprecation-note.md](https://github.com/MicrosoftDocs/msteams-docs/blob/13a5f28497bac343f32ce6719e12c6d90f202c98/msteams-platform%2Fincludes%2Fdeprecation-note.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0913b741-0747-4f4b-9654-1b26ab9b60b3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/deprecation-note.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7ee2b8130a9187e06f399843101ba0de7d1ad7c8/msteams-platform%2Fincludes%2Fdeprecation-note.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/78a7380e-7777-4b7c-a809-5b46888d50d3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/deprecation-note.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c34edd14c9a65f5717cb5476954bf1152246d601/msteams-platform%2Fincludes%2Fdeprecation-note.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5543cf1f-6f63-4709-9f2c-7c463d3bfbfd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/deprecation-note.md](https://github.com/MicrosoftDocs/msteams-docs/blob/beacac62be4f0aba6444117a9c4886abcab79c97/msteams-platform%2Fincludes%2Fdeprecation-note.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b6723a5a-21cf-4431-88e6-c6ac7c9c43ac) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ConnorR777 — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**